CHANGELOG.md

```diff
@@ -16,6 +16,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased 2.x](https://github.com/opensearch-project/neural-search/compare/2.18...2.x)
 ### Features
+- Introduces ByFieldRerankProcessor for second level reranking on documents (This is done locally) ([#932](https://github.com/opensearch-project/neural-search/pull/932))
```

Comment:
@martin-gaievski should we add it in the changelog of main branch now as @brianf-aws already added in the release notes of 2.18?

Reply:
Hey @brianf-aws don't add change in changelog.md as the release notes are already cut.

Reply:
Just keep the change in release notes 2.18. file.

Reply:
Understood will remove from changelog

Reply:
> @martin-gaievski should we add it in the changelog of main branch now as @brianf-aws already added in the release notes of 2.18?

we can skip changelog if we do have entry in the release notes